keps/sig-node/3545-improved-multi-numa-alignment/README.md

```diff
@@ -434,14 +441,19 @@ No.
 
 No.
 
+###### Can enabling / using this feature result in resource exhaustion of some node resources (PIDs, sockets, inodes, etc.)?
+
+No.
+
 ### Troubleshooting
 
 ###### How does this feature react if the API server and/or etcd is unavailable?
+
 N/A.
 
 ###### What are other known failure modes?
 
-TBD.
+No known failure modes.
```

Comment:
On Line 372 under the rollout / rollback section, this is mentioned:

> Kubelet may fail to start. The kubelet may crash.

Is that statement valid, and if so could we identify what those failure modes might be? How does someone recover from that failure mode?

Reply:
Thanks for pointing that out. I think there are only 2 scenarios where kubelet can crash due to this feature:
- bad policy option name, we are already logging appropriate logs for this, to recover one just have to provide correct policy name or disable `TopologyManagerPolicyOptions` 
- cadvisor is not exposing distances for `NUMA` domains, we are also logging it, to recover one has to disable `TopologyManagerPolicyOptions`

I'll update KEP with those steps.

Reply:
Section updated.